setup.py

```diff
@@ -31,7 +31,13 @@
 
 install_requires = [
     'amqp>=1.4.9,<2.0',
+    'Flask>=0.11.1',
+    'Flask-Admin>=1.4.2',
+    'Flask-Breadcrumbs>=0.3.0',
+    'Flask-Script>=2.0.5',
+    'Flask-CLI>=0.4.0',
```

Comment:
> It's required by a lot of other packages anyway

Yes, but we are going to remove this dependency and your overlay should not depend on it directly. The same applies to most of Flask* dependencies in your setup.py. Pin only top-level dependencies and use `pip-tools` to compile your full production requirements.